### DISCUSSION TITLE
Files/Folders sorting

### DISCUSSION THREAD

**Discussion by PluginHunter** [Zettlr/Zettlr#2343] [General]
Hi,

is there a way to change the sorting order for folders/files. I'd like to see directories first and then files (I use the "Combined" file manager mode)

thanks

**Comment by nathanlesage** [comment]
Right now not; it's hard-coded that there are first files and then folders, which has historic reasons. It *might* be easy to add an option for that (but there could be implications I don't forsee right now).

**Reply by PluginHunter** [reply]
Thanks a lot. I would love to have the possibility to choose the order rules. I will raise an enhacement request.

**Reply by nathanlesage** [reply]
No need to open an issue, I just did: #2344 – GitHub is really handy in this regard!

**Reply by PluginHunter** [reply]
Thanks, you're a star
